tests/spec/w3c/headers-spec.js

```diff
@@ -1241,18 +1322,6 @@ describe("W3C — Headers", () => {
         /XXX\s+&\s+the\s+Contributors\s+to\s+the/
       );
     });
-    it("takes additionalCopyrightHolders into account when spec is unofficial", async () => {
```

Comment:
Could this be a breaking change? I'm not sure I follow the rationale behind this.

Reply:
`additionalCopyrightHolders` was being used incorrectly here. It was being used as equivalent of the  `replaceCopyright` option. I think that was a oversight - specially for "unofficial" docs. I checked specs using `additionalCopyrightHolders` and none are "unofficial".